modules/exploits/windows/fileformat/homm3_h3m.rb

```diff
@@ -0,0 +1,175 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+require 'enumerator'
```

Comment:
It was intended for the substring_pos function, but it seems you are correct that it is in fact not required.